k8s/aat/common/ccpay/ccpay-payment-app.yaml

```diff
@@ -0,0 +1,59 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: ccpay
+  namespace: fees-pay
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: ccpay
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: payment-api
+    version: 0.1.0
+  values:
+    rollback:
+      enable: true
+    java:
+      replicas: 2
+      image: hmctspublic.azurecr.io/ccpay/payment-api:prod-*
```

Comment:
This doesn't look right to me. Could you please check if this is correct? IMHO it should be:

`hmctspublic.azurecr.io/payment/api:prod-*`